### DISCUSSION TITLE
`handleInlinePublishForm` makes 'dirty' model attributes for relations.

### DISCUSSION THREAD

**Discussion by TheBnl** [statamic-rad-pack/runway#544] [Help]
Line 240: 
https://github.com/statamic-rad-pack/runway/blob/9ac7f65e4da0866255bd40ebcaf3950369d1fcbd/src/Http/Controllers/CP/ResourceController.php#L240

Sets relations as attributes on a model. This causes the save action to fail because the model tries to store these attributes in the database, so an SQL exception is thrown.

Omitting this method fixes the issue, but it is probably there for a good reason 😉

**Comment by duncanmcclean** [comment]
Usually, this happens when you have fields in your blueprint which don't exist as columns in the database. In this case, you should likely mark those fields as computed:

```yaml
visibility: computed
```

**Reply by TheBnl** [reply]
Hi Duncan, 

I'm not sure i understand correctly, this method sets the handle of a `HasManyFieldType` or `BelongsToFieldtype`  as an attribute before saving the Model to the DB. Because relations are no attributes the error is thrown.

Are you suggesting to set `visibility: computed` on the `HasManyFieldType` blueprint definition?

**Comment by duncanmcclean** [comment]
Sorry, I originally mis-read your issue and converted it to a discussion thinking it was a question of some kind (it didn't follow the bug report template so I assumed it was in the wrong place).

Anyways, should be fixed in #547, which will be tagged shortly.
